### ISSUE TITLE
cannot attach plain text files: unsupported file type

### ISSUE THREAD

**Issue by robbiemu** [ollama-webui/ollama-webui#238]
**Describe the bug**
it seems that I cannot attach text/plain files (saying "unsupported file type 'text/plain'") nor my rust source files (which say "unsupported file type ''").

**To Reproduce**
1. click the paperclip icon in chat
2. select the file and click 'Open'

**Expected behavior**
file should attach

**Screenshots**
<img width="751" alt="Captura de Tela 2023-12-17 às 5 03 13 PM" src="https://github.com/ollama-webui/ollama-webui/assets/248927/a139ceb7-a82c-4417-8d9e-af05a91161fe">

**Desktop (please complete the following information):**
 - OS: MacOs 14.2
 - Browser: chrome
 - Version v1.0.0-alpha.21

additional context:
this was done locally rather than a docker container. but, I encountered no issues following the instructions
I see no error in the console when I do this

**Comment by tjbck** [comment]
Hi, we currently only support image attachment for multi-modal models, but stay tuned for RAG feature #31 after #216. Thanks! Duplicate of  #31
